infraestructure/router/router.go

```diff
@@ -0,0 +1,5 @@
+package router
+
+var SearchPokemon = "/buscarPokemon"
```

Comment:
in router.go depending on which package you use mux, echo or HTTP in your case, you would put the handlder creation here :)